lizmap/modules/admin/zones/project_list.zone.php

```diff
@@ -149,6 +149,15 @@ protected function _prepareTpl()
      */
     private function getProjectListItem($inspectionDirectoryPath, $projectMetadata)
     {
+        $services = lizmap::getServices();
+        $rootRepositories = $services->getRootRepositories();
+        $repository = lizmap::getRepository($projectMetadata->getRepository())->getOriginalPath();
+        if ($rootRepositories != '' && strpos($repository, $rootRepositories) === 0) {
+            $repository = str_replace($rootRepositories, '', $repository);
+            $repository = trim($repository, '/');
+            $repository .= '/';
```

Comment:
if repository can only be a direct child of rootRepository , a dedicated php function fits

```suggestion
            $repository .= basename($repository);
```